StreamConsumer/README.md

```diff
@@ -213,8 +212,7 @@ Rapid pro requires a specific format for phone numbers and it's better to change
   "const": {
     "rapidpro_flow": "https://api.textit.in/api/v2/flow_starts.json",
     "rapidpro_auth_header": "Token API_KEY_GOES_HERE",
-    "rapidpro_flow_id__contact": "4f9a2cbb-2eee-435d-8dca-9505d9ab7b91",
-    "sms_msg": "You have been registered to be contacted on this number."
```

Comment:
@shawnsarwar  the sms body is now coming from RapidPro?

Reply:
Ah yes, this should be updated to reading from `$.source.message.sms_body`